dashboard/app/bisect_test.go

```diff
@@ -390,7 +390,7 @@ Date:   Wed Feb 9 04:05:06 2000 +0000
     kernel: add a fix
 
 bisection log:  %[2]v
-start commit:   11111111 kernel_commit_title1
+latest commit:   11111111 kernel_commit_title1
```

Comment:
Please keep the table properly aligned. This was noted as an important thing by some kernel developers.